src/transaction/makeTransferTransaction.js

```diff
@@ -2,6 +2,32 @@ import makeInputTemplate from './makeInputTemplate'
 import makeTransaction from './makeTransaction'
 
 
+// TODO: Can we remove `export` here somehow, but still be able to import the
+// function for tests?
+export function _makeTransferTransaction(
```

Comment:
I just noticed that I should maybe not do this whole private function thing.
I did it initially for testing, however I should use mocks.